server2/src/main/java/se/salt/server2/domain/account/controller/AccountController.java

```diff
@@ -17,6 +16,7 @@
 @RestController
 @RequestMapping("api/v2/accounts")
 @RequiredArgsConstructor
+@CrossOrigin("*")
```

Comment:
Remind me to set up a custom CORS config with you guys

Reply:
I've also thought about it with a web config. can i send you what I've previously done so you can check if it is a good approach?